tsconfig.json

```diff
@@ -8,13 +8,17 @@
     "noLib": false,
     "preserveConstEnums": true,
     "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "lib": [ "es6" ],
     "suppressImplicitAnyIndexErrors": true,
     "outDir": "./js"
   },
   "types": [
       "node"
   ],
   "include": [
-    "./ts/**/*.ts"
+    "./ts/**/*.ts",
+    "./test/*.ts"
```

Comment:
Including tests here is what creates the `ts/` subfolder and forces `metricly.js` under another folder. You can run the tests in TypeScript, so you don't need to precompile them by updating the `mocha.opts` to point to the TS files. I personally think that's cleaner and you don't have to keep compiling the tests before testing. Gets rid of the `pretest` and `prepare` scripts in `package.json`, too.

Reply:
Agreed. I think this takes care of it https://github.com/metricly/metricly-cli/pull/30/commits/ebbd0ae02a93bfff4fb7016c98bb084bc2c69d9f